script/main.js

```diff
@@ -18,3 +18,92 @@ menuItem2.addEventListener('click', collapse);
 
 const menuItem3 = document.querySelector('#contactMe');
 menuItem3.addEventListener('click', collapse);
+
+const projects = [
+  {
+    title: 'Multi Post Stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply ummy text of the printing and typesetting industry. Lorem Ipsum has been the standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+    mobDescription: 'Lorem Ipsum is simply dummy text of the printing and Ipsum is simply ummy text of the printing and typesetting industry. Lorem Ipsum has been the standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply ummy text of the printing and typesetting industry.',
+    image: 'images/Snapshoot-Portfolio.png',
+    mobImage: 'images/Snapshoot-mobile.png',
+    featureList: ['html', 'bootstrap', 'Ruby'],
+    seeLive: 'https://ifzyy.github.io/My-portfolio/',
+    seeSource: 'https://github.com/ifzyy/My-portfolio',
+  },
+];
+
+const popupWindow = document.querySelector('#extended');
+function popup() {
+  popupWindow.classList.toggle('extended');
+  document.querySelector('#mobileh5').innerHTML = projects[0].title;
+  document.querySelector('.popup-text').innerHTML = projects[0].mobDescription;
+  document.querySelector('#mobimage').src = `${projects[0].mobImage}`;
+  document.querySelector('#seeMobLive').href = projects[0].seeLive;
+  document.querySelector('#seeMobSource').href = projects[0].seeSource;
+}
```

Comment:
- [ ] kindly consider the projects section is created dynamically with the information from the projects array, you are using the same project in all work sections. Please add more projects and add them dynamically.
- [ ] Also consider to add model in dynamic way using JavaScript not same models in all projects
> For more explanation look at [loom video](https://www.loom.com/share/e259d7c756bd4f03ad3592d9e049d49e)